src/styles.scss

```diff
@@ -7,6 +7,10 @@ body {
 
 * {
```

Comment:
formatting is off in this file. Please review.

Reply:
Fixed the styling. FYI I wasn't the one that messed it up.

Reply:
I understand, it was very noticeable so pointed it out.
Please insert a newline after the imports too.